lib/generators/sufia/templates/workflow.json.erb

```diff
@@ -0,0 +1,60 @@
+{
+  "workflows": [
+    {
+      "name": "default",
+      "label": "Digital collections workflow",
+      "description": "A multi-step workflow for digital collections",
+      "actions": [
+         {
+           "name": "finalize_digitization",
```

Comment:
@jcoyne We generate the new workflow only to add a notification which lives in the Sufia namespace Do you think this is the wrong approach?